app/src/main/java/com/babylon/wallet/android/presentation/ui/composables/SimpleAccountCard.kt

```diff
@@ -97,13 +102,16 @@ fun SimpleAccountCard(
             overflow = TextOverflow.Ellipsis
         )
         ActionableAddressView(
-            address = account.address,
+            address = remember(account.address) {
+                Address.Account(AccountAddress.init(account.address))
```

Comment:
this could also be extension function with `Account` receiver

Reply:
Same answer as above